README.rst

```diff
@@ -126,6 +126,44 @@ Run migration
 This command will run the synchronization process between Parse and Syncano. Sit comfortably in your chair and read
 the output.
 
+Syncano Hosting
+---------------
+
+Syncano Hosting is a simple way to host the static files. CLI supports it in the following way:
+
+
+::
+
+    syncano hosting <instance_name> --list
+
+This command will list available hosting in instance.
+
+::
+
+    syncano hosting <instance_name> --list-files <domain>
```

Comment:
I think that for now we should hide <domain> parameter as it doesn't make sense just yet